### PR TITLE
Update advanced.md

### PR DESCRIPTION
**Pull request by shashank-p**
In following link https://tachiyomi.org/help/guides/settings/advanced/#refresh-library-metadata Refresh library metadata genre is mentioned as green.

### CODE REVIEW

File: src/help/guides/settings/advanced.md [hunk 1]
```diff
@@ -1,8 +1,3 @@
----
-title: Advanced
-lang: en-US
----
-
```
Comment (Soitora) [inline src/help/guides/settings/advanced.md:5]: Wait, why was these removed?

Comment (shashank-p) [reply src/help/guides/settings/advanced.md:5]: i thought of it as unnecessary it's my first time working on this repo

Comment (Soitora) [reply src/help/guides/settings/advanced.md:5]: It's needed for our code backend, it's "required" on every page to set page information

Comment (shashank-p) [reply src/help/guides/settings/advanced.md:5]: I'll add it then
